RELEASE-NOTES.md

```diff
@@ -14,5 +14,12 @@
 * `snow snowpark registry` was replaced with `snow registry` command.
 
 ## New additions
+* Builtin plugins
```

Comment:
I think this is not necessary, it's rather implementation detail

Reply:
removed